lib/Screens/light_sensor.dart

```diff
@@ -0,0 +1,10 @@
+import 'dart:async';
+import 'package:light/light.dart';
+
+class LightSensor {
```

Comment:
is a new class for this necessary ? it's only a few lines
I would put this function below the getCamera() one in task widget or maybe even better to put getCamera() in LightSensor class and rename it to something like camera class